src/SV0001Itinerary/Utils/api/DB0002Itineraries.ts

```diff
@@ -0,0 +1,45 @@
+import db from '../fireB/firestore'
+import { doc, collection, getDoc, setDoc, addDoc, deleteDoc, getDocs } from 'firebase/firestore';
+import type { DB0002ItineraryType } from 'SV0001Itinerary/Utils/types/DB0002ItineraryType';
+import { initItinerary } from 'SV0001Itinerary/Utils/types/DB0002ItineraryType';
```

Comment:
↓みたいにして、ドメイン（Itinerary）消せないかな（消せたら9行目以下全コピーでapiファイルは出来るから超楽（一応元のファイルはそうしてた））
```suggestion
import type { DB0002ItineraryType as collectionType } from 'SV0001Itinerary/Utils/types/DB0002ItineraryType';
import { initItinerary as initCollection } from 'SV0001Itinerary/Utils/types/DB0002ItineraryType';
```

Reply:
@Ayato-kosaka 
対応済み
[334c0ac](https://github.com/Ayato-kosaka/react-itinerary/pull/60/commits/334c0acf009b383dc885fb146b15d3b41d489d66)